src/main/kotlin/no/nav/medlemskap/common/Metrics.kt

```diff
@@ -170,14 +170,14 @@ fun apiCounter(): Counter = Counter
 fun clientTimer(service: String?, operation: String?): Timer =
         Timer.builder("client_calls_latency")
                 .tags("service", service ?: "UKJENT", "operation", operation ?: "UKJENT")
-                .description("latency for calls to other services")
+                .description("latency for calls to other clients")
```

Comment:
var det meningen å bytte denne?

Reply:
Hoho, bra catcha! Nei, dette var Intellij som gjorde en search-replace for meg, og som åpenbart var litt triggerhappy...

Reply:
Fikset